packages/editor-ui/src/components/RunDataSchema.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+	<div :class="$style.schemaWrapper">
+		<draggable
+			type="mapping"
+			targetDataKey="mappable"
+			:disabled="!mappingEnabled"
+			@dragstart="onDragStart"
+			@dragend="onDragEnd"
+		>
+			<template #preview="{ canDrop, el }">
+				<div v-if="el" :class="[$style.dragPill, canDrop ? $style.droppablePill : $style.defaultPill]" v-html="el.outerHTML" />
+			</template>
+			<template>
+				<div :class="$style.schema">
+					<run-data-schema-item
+						:schema="schema"
+						:level="0"
+						:parent="null"
+						:subKey="`${schema.type}-0-0`"
+						:mappingEnabled="mappingEnabled"
+						:draggingPath="draggingPath"
+						:distanceFromActive="distanceFromActive"
+						:node="node"
+					/>
+				</div>
+			</template>
+		</draggable>
+	</div>
+</template>
+<script lang="ts" setup>
```

Comment:
nice.. but should this be above template for best practices?

Reply:
same question for other files

Reply:
sure, swapped